aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Merger.java

```diff
@@ -180,22 +180,4 @@
             }
         }, failure -> onNotPresent.call());
     }
-
-    /**
-     * The strategy to use while merging. Whether to consider the contents of the mutation
-     * outbox before saving data locally, or, to ignore it.
-     */
-    enum MergeStrategy {
-        /**
-         * When merging, the contents of the mutation outbox will *not* be considered.
-         */
-        IGNORE_PENDING_MUTATIONS,
-
-        /**
-         * When merging, the contents of the mutation outbox will be considered.
-         * If there is already a pending mutation in the mutation outbox, for a model of the
-         * same ID as the model being merged -- then the merge will *not* modify the existing model.
-         */
-        CONSIDER_PENDING_MUTATIONS
-    }
```

Comment:
where was this being used before?

Reply:
There use to be some logic to change the behavior depending on whether the change was outbound, or inbound.

If outbound, the mutation outbox would have a pending mutation, while being merged. So, we'd have to not consider it.

The issue was later remedied by changing the sequence of events, so that the event gets removed from the outbox before being merged.

But, this enum apparently didn't removed in that previous change.